README.md

```diff
@@ -235,6 +235,55 @@ set(this, 'metrics.context.userName', 'Jimbo');
 this.metrics.trackPage({ page: 'page/1' }); // { userName: 'Jimbo', page: 'page/1' }
 ```
 
+## Octane / Native Class usage
+
+If you are living on the edge and using an app built with the [Ember Octane Blueprint](https://github.com/ember-cli/ember-octane-blueprint) or otherwise implementing Native Class syntax in your routes, the following example can be used to report route transitions to ember-metrics:
+
+```js
+// app/routes/application.js
+import Route from '@ember/routing/route'
+import { action } from '@ember/object'
+import { inject as service } from '@ember/service'
+import { scheduleOnce } from '@ember/runloop'
+
+export default class ApplicationRoute extends Route {
+  @service metrics
+  @service router
+
+  @action
+  didTransition() {
+    this._super(...arguments)
```

Comment:
should it be 
```js
didTransition() {
     super.didTransition(...arguments);
}

```

?

Reply:
I think you're right. I'll test it to be sure.

Reply:
changing
`this._super(...arguments);`
to
`super.didTransition(...arguments);`

yields this error:
```
TypeError: (intermediate value).didTransition is not a function
    at ApplicationRoute.didTransition (/tmp/broccoli-28387PLcPVSpmiYuJ/out-275-broccoli_merge_trees/assets/aliencyborg/routes/application.js:28:1)
    at Class.triggerEvent (/tmp/broccoli-28387PLcPVSpmiYuJ/out-275-broccoli_merge_trees/assets/@ember/-internals/routing/lib/system/router.js:1290:1)
...
```